README.md

```diff
@@ -1,77 +1,86 @@
-The Fintoc Quickstart repository has a frontend application written in React, which integrates the Widget, as well as multiple backend applications that use the languages supported by the Fintoc SDKs.
+**Fintoc Quickstart** is the repository that allows you to test Fintoc tools locally!
+The purpose of the repository is that you know how to integrate the Fintoc Widget and the SDKs that we offer for the different supported languages.
+**Fintoc Quickstart** is composed mainly by what we call a frontend application, which is an application written in [React](https://reactjs.org/) and that integrates the Fintoc [Widget](https://docs.fintoc.com/docs/usando-el-widget#flujo-del-widget), and also by backend applications which represent the backend and that use the SDKs that Fintoc provides. 
+Currently the languages supported by the Fintoc SDKs are: [**Node.js**](https://github.com/fintoc-com/fintoc-node), [**Ruby**](https://github.com/fintoc-com/fintoc-ruby) and [**Python**](https://github.com/fintoc-com/fintoc-python), but don't worry, we are working to support others!
+
+<p align="center">
+  <img src="_media/fintoc.png" />
+</p>
+
+# Table of content
+* [Fintoc Quickstart](#fintoc-quickstart)
+* [Table of content](#table-of-content)
+* [1. Clone the repository](#1-clone-the-repository)
+* [2. Get your keys and setup](#2-get-your-keys-and-setup)
+* [3. Run Fintoc Quickstart](#3-run-fintoc-quickstart)
+    * [3.1. Expose backend to internet (ngrok)](#31-expose-backend-to-internet-ngrok)
+    * [3.2. Run](#32-run)
+        * [Prerequisites](#prerequisites)
+        * [Commands](#commands)
+            * [Frontend](#frontend)
+            * [Node](#node)
+            * [Python](#python)
+            * [Ruby](#ruby)
+    * [3.3. Run + Docker](#33-run--docker)
+        * [Prerequisites](#prerequisites-1)
+        * [Commands](#commands-1)
+            * [help](#help)
+            * [start](#start)
+            * [build](#build)
+            * [destroy](#destroy)
+            * [stop](#stop)
+            * [restart](#restart)
+            * [logs](#logs)
 
 # 1. Clone the repository
-Clone the repository using your prefered terminal:
-<table>
-<tr>
-<th>Protocol</th>
-<th>Command</th>
-</tr>
-<tr>
-<td>HTTPS</td>
-<td>
-<pre lang="shell">
-$ git clone https://github.com/fintoc-com/quickstart.git
-</pre>
-</td>
-</tr>
-<tr>
-<td>SSH</td>
-<td>
-<pre lang="shell">
-$ git clone git@github.com:fintoc-com/quickstart.git
-</pre>
-</td>
-</tr>
-<tr>
-<td>GitHub CLI</td>
-<td>
-<pre lang="shell">
-$ gh repo clone fintoc-com/quickstart
-</pre>
-</td>
-</tr>
-</table>
-
-And enter it with:
+Clone the repository using your prefered terminal using any of the following commands:
+
+| Protocol   | Command                                                        |
+|------------|----------------------------------------------------------------|
+| HTTPS      | ```$ git clone https://github.com/fintoc-com/sample-app.git``` |
```

Comment:
Here the url should be quickstart right?

Reply:
You're right, i fix it now